### PR TITLE
created fix to disable sound when unavailable

### PR DESCRIPTION
**Pull request by DecryptingElectrons**
this is a fix for issue #57

side note, for testing.. this issue is reproducible by deleting a sound file before compilation.

### CODE REVIEW

File: src/main/java/globalquake/sounds/Sounds.java [hunk 1]
```diff
@@ -52,6 +56,12 @@ public static void load() throws Exception {
 		felt = loadSound("sounds/felt.wav");
 		dong = loadSound("sounds/dong.wav");
 
+		if(!soundsAvailable)
+		{
+			Logger.info("An error occurred while loading sounds. Sounds will be disabled.");
+			return;
+		}
+
```
Comment (xspanger3770) [inline src/main/java/globalquake/sounds/Sounds.java:64]: This is not needed if the exception is thrown in the first loadSound call

File: src/main/java/globalquake/ui/globalquake/GlobalQuakePanel.java [hunk 2]
```diff
@@ -110,7 +111,16 @@ private void drawTexts(Graphics2D g) {
         g.drawString(str, getWidth() - g.getFontMetrics().stringWidth(str) - 6, getHeight() - 9);
 
         List<String> settingsStrings = new ArrayList<>();
-        settingsStrings.add("Sound Alarms: %s (S)".formatted(Settings.enableSound ? "Enabled" : "Disabled"));
+
+        //If sound is not available, set a special message
+        if(!Sounds.soundsAvailable)
+        {
+            settingsStrings.add("Sound Alarm unavailable: See log");
```
Comment (xspanger3770) [inline src/main/java/globalquake/ui/globalquake/GlobalQuakePanel.java:118]: Just a detail.. but I would let the message say "Sound Alarms unavailable" as the user would already see the error popup during the loading process

**Comment by xspanger3770** [comment]
Really appreciate your help! my suggestions are in the code review.

**Comment by DecryptingElectrons** [comment]
I made the suggested changes. I also added a warning dialog in the exception handler. It seemed better to spawn the dialog box as a warning instead of an error.

**Pull request merged**